tools/influx_si/influx_si.xml

```diff
@@ -1,232 +1,62 @@
-<tool id="influx_si" name="influx_si" version="@TOOL_VERSION@+galaxy1">
-   <description>metabolic flux analysis</description>
-   <macros>
-        <token name="@TOOL_VERSION@">5.1.0</token>
-   </macros>
-   <requirements>
-      <requirement type="package" version="@TOOL_VERSION@">influx_si</requirement>
-   </requirements>
-   <version_command>influx_s --version</version_command>
-   <command detect_errors="exit_code"><![CDATA[
-       #set $inp_all = [$input_main, ($prlexp.input_aux if $prlexp.input_aux else [])]
-       #if $si.s_i == 'i':
-         #silent $inp_all.append($si.input_ti)
-       #end if
+<tool id="influx_si" name="influx_si" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.09">
+    <description>
+        Estimate metabolic fluxes and concentrations by fitting simulated labeling in metabolites to NMR/MS measurements
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">influx_si</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir outdir;
```

Comment:
please use && instead of ;

Reply:
In fact,  I used ";" to keep a possibility to postpone returning the resulting code of influx_si in case of error, to prepare outdir with files which could be helpful to the end-user to understand what went wrong and why. Now, with your remark, I implemented this, cf 'exit $ret'. That's why I keep ";".